src/components/AuthenticatedAppHeader/SightingsButton.jsx

```diff
@@ -0,0 +1,81 @@
+import React, { useState } from 'react';
+
+import List from '@material-ui/core/List';
+import Divider from '@material-ui/core/Divider';
+
+import useSightingTermQuery from '../../models/sighting/useSightingTermQuery';
+import Text from '../Text';
+import SearchButton from './SearchButton';
+import SearchResult from './SearchResult';
+
+export default function SightingsButton() {
+  const [inputContent, setInputContent] = useState('');
+  const [searchTerm, setSearchTerm] = useState('');
+
+  const {
+    data: searchResults,
+    loading,
+    error,
+  } = useSightingTermQuery(searchTerm);
+
+  const resultsCurrent = inputContent === searchTerm;
+  const noResults = searchResults && searchResults.length === 0;
+  const mappableSearchResults = resultsCurrent
+    ? searchResults || []
```

Comment:
I think parentheses here to disambiguate would be helpful.

Reply:
the formatter decides on the parentheses =/